Orange/widgets/visualize/tests/test_owsilhouetteplot.py

```diff
@@ -1,3 +1,4 @@
+# pylint disable=protected-access
```

Comment:
Sorry for my typeo: this has no effect due to a missing a colon after pylint (see below). You can also add a comment `# Test methods are allowed to access private members` above.